MapboxNavigation/FeedbackViewController.swift

```diff
@@ -165,10 +178,48 @@ class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecog
         progressBar.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
         progressBar.bottomAnchor.constraint(equalTo: view.safeBottomAnchor).isActive = true
     }
+    
+    func defaultFeedbackHandlers(source: FeedbackSource = .user) -> (send: (FeedbackItem) -> Void, dismiss: () -> Void) {
+        let uuid = eventsManager.recordFeedback()
+        let send = defaultSendFeedbackHandler(uuid: uuid)
+        let dismiss = defaultDismissFeedbackHandler(uuid: uuid)
+        
+        return (send, dismiss)
+    }
+    
+    func defaultSendFeedbackHandler(source: FeedbackSource = .user, uuid: UUID) -> (FeedbackItem) -> Void {
+        return { [weak self] (item) in
+            guard let strongSelf = self else { return }
+            
+            // todo, can this be moved as a delegate on this view controller?
+            //strongSelf.delegate?.mapViewController(strongSelf, didSendFeedbackAssigned: uuid, feedbackType: item.feedbackType)
```

Comment:
Would it make sense to make a protocol for FeedbackViewController here? This would remove some plumbing.

Reply:
Is this delegate method even helpful? It only returns the uuid and the feedback type.

Reply:
To make it useful, pass in the feedback item itself instead of the feedback type.